Prefer concat in samples #2537 (#2544)
akka/akka-http: docs/src/main/paradox/routing-dsl/directives/index.md

@@@ div { .group-scala }

### Composing Directives with `~` Operator

@@@ 

@@@ note { .group-scala }
Gotcha: forgetting the `~` (tilde) character in between directives can result in perfectly valid

> Scala code that compiles but does not work as expected. What would be intended as a single expression would actually be multiple expressions, and only the final one would be used as the result of the parent directive. Because of this, the recommended way to compose routes is with the the `concat` combinator.

Reviewer: :+1:

@@@

@@@ div { .group-scala }

> Alternatively we can combine directives using `~` operator where we pass each directive as an argument to the combinator function instead of chaining them with `concat` . Let's take a look at the usage of this combinator:

Reviewer: ```suggestion
Alternatively we can combine directives using the `~` operator where we chain them together instead of passing each directive as a separate argument. Let's take a look at the usage of this combinator:
```
